flow/Knobs.cpp

```diff
@@ -74,7 +74,8 @@ void FlowKnobs::initialize(Randomize randomize, IsSimulated isSimulated) {
 
 	init( WRITE_TRACING_ENABLED,                              true ); if( randomize && BUGGIFY ) WRITE_TRACING_ENABLED = false;
 	init( TRACING_SAMPLE_RATE,                                 0.0 ); // Fraction of distributed traces (not spans) to sample (0 means ignore all traces)
-	init( TRACING_UDP_LISTENER_PORT,                          8889 ); // Only applicable if TracerType is set to a network option
+	init( TRACING_UDP_LISTENER_ADDR,                          "127.0.0.1"); // Only applicable if TracerType is set to a network option
```

Comment:
Nit: fix the spacing here so the closing paren matches the others (clang-format does not modify the knobs)

Reply:
huh, figured you could rely on the formatter here.

Reply:
There's a `// clang-format off` comment near the top, I think the formatter was wanting to modify things in this file a bit too much so it was disabled :)

Reply:
It still looks off (when I click View File).

Reply:
The ws is there now.